lib/iStats/smc.rb

```diff
@@ -122,9 +125,29 @@ def scan_supported_keys
         puts "The enabled sensors will show up when running `istats` or `istats extra`."
       end
 
+      def zabbix_discover
+        items = []
+
+        characters = [('a'..'z'), ('A'..'Z'),(0..9)].map { |i| i.to_a }.flatten
+        characters.each {|l1|
+          characters.each {|l2|
+            characters.each {|l3|
+              key = "T#{l1}#{l2}#{l3}"
+              if (name(key) != 'Unknown') && (name(key) != '')
+                t = is_key_supported(key);
+                if (t > 0) 
+                    items.push('{"{#KEY}":"' + key + '","{#NAME}":"' + name(key) + '" }')
```

Comment:
Is that really the best way to build a JSON object in Ruby? Looks a bit hacky with string concatenation.